project-base/storefront/helpers/filterOptions/seoCategories.ts

```diff
@@ -214,3 +216,20 @@ export const getFilterWithoutSeoSensitiveFilters = (
 
     return { filteredFilter, filteredSort };
 };
+
+export const useRedirectFromSeoCategory = () => {
+    const setWasRedirectedFromSeoCategory = useSessionStore((s) => s.setWasRedirectedFromSeoCategory);
+
+    const redirectFromSeoCategory = (
+        pushQueryFilter: (
+            newFilter?: FilterQueries,
+            pathnameOverride?: string,
+            sortOverride?: ProductOrderingModeEnumApi,
```

Comment:
If I understand the logic correctly this should be simple type of `pushQueryFilter: () => void`.

Reply:
You are absolutely correct. Fixed.